engine/engine/src/engine.cpp

```diff
@@ -106,6 +104,9 @@ extern "C" {
 
 namespace dmEngine
 {
+
+    using namespace dmVMath;
```

Comment:
In cpp files, we may open up the namespace if there are a lot of uses of the types/functions.